### PR TITLE
Removed Dterm Rpm and Vbat Pid Compensation header options from 4.3

### PR DESCRIPTION
**Pull request by Asizon**
To go with https://github.com/betaflight/betaflight/pull/9633 and https://github.com/betaflight/betaflight/pull/9984

### CODE REVIEW

File: js/header_dialog.js [hunk 1]
```diff
@@ -83,9 +84,9 @@ function HeaderDialog(dialog, onSave) {
             {name:'gyro_rpm_notch_harmonics'    , type:FIRMWARE_TYPE_BETAFLIGHT,  min:'4.1.0', max:'999.9.9'},
             {name:'gyro_rpm_notch_q'            , type:FIRMWARE_TYPE_BETAFLIGHT,  min:'4.1.0', max:'999.9.9'},
             {name:'gyro_rpm_notch_min'          , type:FIRMWARE_TYPE_BETAFLIGHT,  min:'4.1.0', max:'999.9.9'},
-            {name:'dterm_rpm_notch_harmonics'   , type:FIRMWARE_TYPE_BETAFLIGHT,  min:'4.1.0', max:'999.9.9'},
-            {name:'dterm_rpm_notch_q'           , type:FIRMWARE_TYPE_BETAFLIGHT,  min:'4.1.0', max:'999.9.9'},
-            {name:'dterm_rpm_notch_min'         , type:FIRMWARE_TYPE_BETAFLIGHT,  min:'4.1.0', max:'999.9.9'},
+            {name:'dterm_rpm_notch_harmonics'   , type:FIRMWARE_TYPE_BETAFLIGHT,  min:'4.1.0', max:'4.2.0'},
+            {name:'dterm_rpm_notch_q'           , type:FIRMWARE_TYPE_BETAFLIGHT,  min:'4.1.0', max:'4.2.0'},
+            {name:'dterm_rpm_notch_min'         , type:FIRMWARE_TYPE_BETAFLIGHT,  min:'4.1.0', max:'4.2.0'},
```
Comment (TheIsotopes) [inline js/header_dialog.js:89]: the same for these three

File: js/header_dialog.js [hunk 2]
```diff
@@ -28,6 +28,7 @@ function HeaderDialog(dialog, onSave) {
             {name:'airmode_activate_throttle'	, type:FIRMWARE_TYPE_BETAFLIGHT,  min:'2.8.0', max:'999.9.9'},
             {name:'rollPitchItermIgnoreRate'	, type:FIRMWARE_TYPE_BETAFLIGHT,  min:'2.8.0', max:'3.0.1'},
             {name:'yawItermIgnoreRate'			, type:FIRMWARE_TYPE_BETAFLIGHT,  min:'2.8.0', max:'3.0.1'},
+            {name:'vbat_pid_compensation'       , type:FIRMWARE_TYPE_BETAFLIGHT,  min:'2.8.0', max:'4.2.0'},
```
Comment (TheIsotopes) [inline js/header_dialog.js:31]: imho it should be 4.2.9 - to include maintenance releases as well

File: js/header_dialog.js (hunk 1 above)
Comment (TheIsotopes) [inline js/header_dialog.js:89]: the same for these three

File: js/header_dialog.js (hunk 2 above)
Comment (Asizon) [reply js/header_dialog.js:31]: Sure @thelsotopes, thanks!

Comment (mikeller) [reply js/header_dialog.js:31]: Good point. Or even 4.2.999 to follow the pattern that is used.

Comment (TheIsotopes) [reply js/header_dialog.js:31]: bad idea ... the pattern is 999.9.9 not 999.9.999

Comment (Asizon) [reply js/header_dialog.js]: yes, 999 is used only for the first number, done!

Comment (mikeller) [reply js/header_dialog.js]: This is broken now - it will fail for 4.2.10 and following. Please learn how to use SemVer.

Comment (Asizon) [reply js/header_dialog.js]: Yes right @mikeller, then im going to use 999 like you suggested

**Pull request merged**